monai/bundle/meta_schema.json

```diff
@@ -0,0 +1,220 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$defs": {
+        "tensor": {
+            "$comment": "Represents a tensor object argument/return value, can be MetaTensor.",
+            "type": "object",
+            "properties": {
+                "type": {
+                    "type": "string"
+                },
+                "format": {
+                    "type": "string"
+                },
+                "num_channels": {
+                    "type": "integer"
+                },
+                "spatial_shape": {
+                    "type": "array",
+                    "items": {
+                        "type": [
+                            "string",
+                            "integer"
+                        ]
+                    }
+                },
+                "dtype": {
+                    "type": "string"
+                },
+                "value_range": {
+                    "type": "array",
+                    "items": {
+                        "type": "number"
+                    }
+                },
+                "is_patch_data": {
+                    "type": "boolean"
+                },
+                "channel_def": {
+                    "type": "object"
+                }
+            },
+            "required": [
+                "type",
+                "format",
+                "num_channels",
+                "spatial_shape",
+                "dtype",
+                "value_range"
+            ]
+        },
+        "argument": {
+            "$comment": "Acceptable input types for model forward pass, arbitrary Python objects not covered.",
+            "anyOf": [
+                {
+                    "type": "number"
+                },
+                {
+                    "type": "integer"
+                },
+                {
+                    "type": "string"
+                },
+                {
+                    "type": "boolean"
+                },
+                {
+                    "$ref": "#/$defs/tensor"
+                }
+            ]
+        },
+        "result": {
+            "$comment": "Return value from a model's forward pass, same as an argument for now.",
+            "$ref": "#/$defs/argument"
+        },
+        "network_io": {
+            "description": "Defines the format, shape, and meaning of inputs and outputs to the model.",
```

Comment:
Do we need to mention `optionally "post_processed_outputs` here, which is mentioned in **network_data_format** section of the mb_spec rst?

Reply:
I'll add something for this.